src/measure_theory/group/measure.lean

```diff
@@ -716,6 +716,21 @@ example {E : Type*} [normed_add_comm_group E] [normed_space ℝ E] [nontrivial E
 
 end
 
+variables [nontrivially_normed_field 𝕜] [topological_space G] [topological_space H]
+  [add_comm_group G] [add_comm_group H] [topological_add_group G] [topological_add_group H]
+  [module 𝕜 G] [module 𝕜 H] (μ : measure G) [is_add_haar_measure μ] [borel_space G] [borel_space H]
+  [t2_space H]
+
+instance map_continuous_linear_equiv.is_add_haar_measure (e : G ≃L[𝕜] H) :
```

Comment:
```suggestion
instance _root_.continuous_linear_equiv.is_add_haar_measure_map (e : G ≃L[𝕜] H) :
```

Reply:
(since that's consistent with the `e.to_add_equiv.is_add_haar_measure_map` dot notation)

Reply:
Would you mind making a fixup?

Reply:
I'm not sure I understand your suggestion, actually. `add_equiv.is_add_haar_measure_map` is a lemma while `map_continuous_linear_equiv.is_add_haar_measure` is an instance

Reply:
Oh, I totally missed it was an instance. The naming seems pretty weird to me still, but instance naming doesn't matter much anyway. No action requested after all :)